src/dropdown/index.js

```diff
@@ -39,9 +37,9 @@ export class Dropdown extends Component {
   };
 
   _renderChildren = (children) => {
-    if (children.length === 0) return null;
+    if (React.Children.count(children) === 0) return null;
 
-    return [].concat(children).map( (opt, i) => {
+    return React.Children.map(children, (opt, i) => {
```

Comment:
Weird, I thought I got this with my previous PR (https://github.com/influitive/patternity/pull/173/files)